src/components/SwitchTheme/hook.ts

```diff
@@ -1,7 +1,7 @@
 import { useState, useEffect, useCallback } from 'react';
 import { THEMES } from 'src/constants';
 
-export const useToggleTheme = () => {
+export const useThemeChange = () => {
```

Comment:
the name of the hook should be exact to the name of component. read article about custom hooks